dynawo/sources/Models/Modelica/Dynawo/Examples/package.order

```diff
@@ -15,4 +15,4 @@ RVS
 SMIB
```

Comment:
Could you also update the examples in OpoenModelica, mainly the SMIB.Standard that uses St4b?

Reply:
The test cases have been updated. Moreover, I have regenerated the .csv for all cases, with the same variables in each file : UPu, EfdPu, PmPu, PGenPu.